tests/api/test_record_relations.py

```diff
@@ -1085,3 +1084,245 @@ def test_siblings_relations(client, json_headers, testdata, users):
         }
     ]
     _test_sibl_invalid_relations_should_fail(client, json_headers, invalids)
+
+
+# Sequence Relations
+def _test_sequence_invalid_relations_should_fail(
+    client, json_headers, invalids, status_code=400
+):
+    """Test relation creation with invalid link relation should fail."""
+    api_endpoint_documents = "invenio_app_ils_relations.docid_relations"
+
+    for invalid in invalids:
+        next_pid = invalid["next_pid"]
+        next_pid_type = invalid["next_pid_type"]
+        previous_pid = invalid["previous_pid"]
+        previous_pid_type = invalid["previous_pid_type"]
+        relation_type = invalid["relation_type"]
+
+        url = url_for(api_endpoint_documents, pid_value=next_pid)
+        payload = {
+            "next_pid": next_pid,
+            "next_pid_type": next_pid_type,
+            "previous_pid": previous_pid,
+            "previous_pid_type": previous_pid_type,
+            "relation_type": relation_type,
+        }
+
+        res = client.post(url, headers=json_headers, data=json.dumps(payload))
+        assert res.status_code == status_code
+        if status_code == 400:
+            error = json.loads(res.data.decode("utf-8"))
+            assert "message" in error
+            assert next_pid in error["message"]
+            assert previous_pid in error["message"]
+
+
+def _test_sequence_document(client, json_headers):
```

Comment:
just about naming, is it more `_test_sequence_series`? And below the same.

Reply:
I have already changed that in the upcoming request, and ported the change in this branch too, I went for `_test_sequence` since its only allowed for series, is redundant to mention it.